Asterix/optics/coronagraph.py

```diff
@@ -578,3 +595,182 @@ def HLC(self):
                 self.EF_from_phase_and_ampl(ampl_abb=ampl_hlc, phase_abb=phase_hlc, wavelengths=wav))
 
         return hlc_all_wl
+
+
+def fqpm_mask(dim):
+    """
+    Create a FQPM phase mask.
```

Comment:
A) that's completely useless and B) I didn't write this function, I just moved it out. I will maintain existing author mentions because I understand it's important to some but I won't be adding mine. Public repo code is searchable and I don't want my name be popping up in random functions.

Reply:
I looked at this again, who would i mark down for this? You and Axel? How do I know who wrote which line? It just messy, so please advise if you want me to add an author here.

Reply:
It's Axel probably. Put the date where you split the function in 2. For the function you put in this PR just put Galicher as author and the date you translated them in Asterix. In general, put major modif to the function and date of change.

I agree that it's useless and messy, it's a way to keep track of modifications pre-git but they are attached to it so I indulge. You can put "ILa" if you don't want your name appearing everywhere.